verispy/veris.py

```diff
@@ -368,7 +370,7 @@ def json_to_df(self, filenames=None, keep_raw=False, schema_path=None, schema_ur
         """
 
         # load schema
-        if verbose: print('Loading schema')
+        if verbose : print('Loading schema')
```

Comment:
PEP 8 says to leave this how I did it

https://www.python.org/dev/peps/pep-0008/#id26